packages/components/scripts/vendorism/custom/alert.vendorism.js

```diff
@@ -25,6 +27,25 @@ const transformStyles = (path, originalContent) => {
   };
 };
 
+const transformComponent = (path, originalContent) => {
+  const content = replaceSections([
+    // Begin remove countdown
+    [
+      "@property({ type: String, reflect: true }) countdown?: 'rtl' | 'ltr';",
+      "countdown? : 'rtl' | 'ltr';",
```

Comment:
I would do it to a `private` property, otherwise it appears in our storybook table under properties: 

![image](https://github.com/user-attachments/assets/6d56d28c-f240-49e5-942b-4d28e469bccd)
```suggestion
      "private countdown? : 'rtl' | 'ltr';",
```

Reply:
Fixed